docs/guides/setup/dpdk-setup-guide.md

```diff
@@ -17,6 +17,9 @@ For build instructions, see [P4 SDE Readme](https://github.com/p4lang/p4-dpdk-ta
 
 ### Install basic utilities
 
+Note: See [OpenSSL security guide](/guides/security/openssl-guide.md)
+for OpenSSL version and EOL information.
+
 ```bash
```

Comment:
It is better practice to separate the introductory text from the code sample:

----

For a Fedora system:
```bash
yum install libatomic libnl3-devel openssl
```
For an Ubuntu system:
```bash
apt install libatomic1 libnl-route-3-dev openssl
```

Reply:
Fixed

Reply:
You fixed it in the es2k setup guide but not in the dpdk setup guide.